ynr/apps/popolo/migrations/0036_auto_20210310_1210.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.2.16 on 2021-03-10 12:10
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("parties", "0012_add_party_docs"),
+        ("popolo", "0035_attach_memberships_to_posts"),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
```

Comment:
This isn't actually part of this PR - there are a few migrations that need applying from changes I made in the past. Can you remove this line, and then we can add the unapplied migrations in a later PR